src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -437,4 +437,16 @@
   <data name="DelegateInvocationCanBeSimplified" xml:space="preserve">
     <value>Delegate invocation can be simplified.</value>
   </data>
+  <data name="UseExplicitTyping" xml:space="preserve">
+    <value>Use explicit typing instead of 'var'</value>
```

Comment:
i would far prefer you make the key:  "Use_explicit_typeing_instead_of_var".  Now i can easily find the key if i know the loc'ed string.

Also, "typing" sounds very weird.  It sounds like you're inflecting the verb "to type".  I'd really like if it just said "Use explicit type instead of 'var'".